tests/e2e/util/statefulset/statefulset.go

```diff
@@ -64,7 +64,7 @@ func IsAllDesiredPodsRunningAndReady(c kubernetes.Interface, sts *appsv1.Statefu
 			return false
 		}
 	}
-	log.Logf("desired pods of sts %s/%s are running and ready (%v)", sts.Namespace, sts.Name, actualPodOrdinals.List())
+	// log.Logf("desired pods of sts %s/%s are running and ready (%v)", sts.Namespace, sts.Name, actualPodOrdinals.List())
```

Comment:
Because this is just one check in the whole `xxMemberReadyFn`, which will output a lot of bothering logs. I added specific logs after `pd`, `tikv`, etc ready instead.